app/views/templates/base.php

```diff
@@ -75,6 +75,7 @@
 } ?><?= $title_productname ?></title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Transvision is a Web application targeting the Mozilla localization community, created and maintained by the French Mozilla community">
```

Comment:
I think we might need to shorten it, usually a description should be around 160 characters or less (unless we don't want to bother about Google)
https://moz.com/blog/i-cant-drive-155-meta-descriptions-in-2015